### PR TITLE
Add statement about browser compatibility to docs.

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

**Comment by hayd** [comment]
I always read "browser compatibility" the other way around, that any js/ts that runs in the browser ought to run in deno (except DOM manipulation etc)...

**Comment by ry** [comment]
> I always read "browser compatibility" the other way around, that any js/ts that runs in the browser ought to run in deno (except DOM manipulation etc)...

There are many APIs that deno does not implement. Like - accessing the web cam ... The opposite is more formal.

File: Docs.md [hunk 1]
```diff
@@ -63,6 +63,17 @@ Or see the [doc website](https://deno.land/typedoc/index.html).
 If you are embedding deno in a Rust program, see
 [the rust docs](https://deno.land/rustdoc/deno/index.html).
 
+## Browser compatibility
+
+Not all Deno programs will run in the browser, but a subset of them can. For
+example, a TypeScript program will not run in the browser without transpiling.
+Therefore it's important to explicitly and complete define what compatibility
```
Comment (jscharf) [inline Docs.md]: Did you mean to say `completely define` here?

Comment (ry) [reply Docs.md]: Fixed.

**Pull request merged**